Week2/homework/maartjes_work.js

```diff
@@ -44,4 +42,22 @@ const tuesday = [
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+const durationsInHours = tasks.map(task => Number(task.duration / 60).toFixed(2));
+
+const twoHrsLessDurations = durationsInHours.filter(twohrs => twohrs >= 2);
+
+const sallaryPerTask = durationsInHours.map(sallaryPerduration => sallaryPerduration * 20);
+console.log(sallaryPerTask);
+
+const totalSallery = sallaryPerTask.reduce((accum, value) => accum + value, 0);
+
+console.log(totalSallery);
+
+
+function formatEuro(money) {
+  return '€ ' + Number(money).toFixed(2)
+}
```

Comment:
This function is a bit overkill. It does next to nothing and it is used once only. You do not need to convert `money` to a number. It should be a number already.